examples/trials/network_morphism/FashionMNIST/config.yml

```diff
@@ -0,0 +1,29 @@
+authorName: default
+experimentName: example_FashionMNIST-network-morphism
+trialConcurrency: 4
+maxExecDuration: 48h
+maxTrialNum: 200
+#choice: local, remote, pai
+trainingServicePlatform: local
+#searchSpacePath: search_space.json
+#choice: true, false
+useAnnotation: false
+tuner:
+  #choice: TPE, Random, Anneal, Evolution, BatchTuner, NetworkMorphism
+  #SMAC (SMAC should be installed through nnictl) 
+  builtinTunerName: NetworkMorphism
+  classArgs:
+    #choice: maximize, minimize
+    optimize_mode: maximize
+    #for now, this tuner only supports cv domain
+    task: cv
+    #input image width
+    input_width: 28
+    #input image channel
+    input_channel: 1
+    #number of classes
+    n_output_node: 10
+trial:
+  command: python FashionMNIST_keras.py
```

Comment:
suggest change  all `python` to `python3` in `config.yml`  to conform with other examples & offical environments

Reply:
good suggestion~

Reply:
Thank you for your suggestion.